### DISCUSSION TITLE
Clicking on Profile picture doesn't display Sign out, Lock, Change account settings, or Switch accounts

### DISCUSSION THREAD

**Discussion by jltayl2** [valinet/ExplorerPatcher#3348] [Q&A] [answered]
When I click on the Start menu, it doesn't give me the option of changing to a different account, signing out, locking the computer, or changing my account settings (See attached screenshot). This means I can't even log off of my computer.
![Start Menu - Accounts](https://github.com/valinet/ExplorerPatcher/assets/46336741/36a34de7-13a1-47c9-bc44-cbdf1a87e5bf)

I've checked all of the setting options, but nothing was applicable to what I'm missing.

**Comment by pyrates999** [comment, marked as answer]
Update to the latest pre-release of EP.

**Reply by jltayl2** [reply]
Thanks! That did the trick!
